src/com/company/Main.java

```diff
@@ -3,6 +3,71 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+        byte clientOS = 0;
+        if (clientOS == 1) {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        } else if (clientOS == 0) {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        }
+
+
+        byte clientOS2 = 1;
+        short clientDeviceYear = 2013;
+        if (clientOS2 == 1) {
+            if (clientDeviceYear >= 2015) {
+                System.out.println("Установите версию приложения для iOS по ссылке");
+            } else System.out.println("Установите облегченную версию приложения для iOS по ссылке");
+        } else if (clientOS2 == 0) {
+            if (clientDeviceYear >= 2015) {
+                System.out.println("Установите версию приложения для Android по ссылке");
+            } else System.out.println("Установите облегченную версию приложения для Android по ссылке");
+        }
+
+        int year = 2021;
+        if (year % 400 == 0) {
+            System.out.println(year + " год является високосным");
+        } else {
+            if (year % 4 == 0 && year % 100 != 0) {
```

Comment:
```suggestion
        } else if (year % 4 == 0 && year % 100 != 0) {
```
смысл в целом не меняется, но я бы продолжал условия в рамках внешнего условного оператора